doc/fgpu.design.rst

```diff
@@ -375,8 +375,10 @@ The remaining steps are to
  3. Do a partial transpose, so that *spectra-per-heap* (256 by default) spectra
     are stored contiguously for each channel (the Nyquist frequencies are also
     discarded at this point).
- 4. Convert to int8.
- 5. Interleave the polarisations.
+ 4. Convert to integer.
+ 5. Where the output bits per sample is not a whole number of bytes, do the
+    necessary bit-packing.
```

Comment:
Is this referring to your `gaussian_dtype` update to `utils.py`?

Reply:
No, I'm referring to [this code](https://github.com/ska-sa/katgpucbf/blob/111229296f30d3e5cd0bd62b849b0d3b5535ab81/src/katgpucbf/fgpu/kernels/postproc.mako#L83-L87) (and the code that calls it).